docs/osf-trivia-list/dasharo.md

```diff
@@ -235,6 +235,81 @@ Dasharo Tools Suite
 * We provide ready to use binaries with GPG based signing scheme that improve
   verification where firmware coming from.
 
+## What are the differences between the official coreboot repository and the Dasharo repository?
+
+* Dasharo focus on specific platforms listed in  [supported
+  hardware](https://docs.dasharo.com/unified/novacustom/overview/) section of Dasharo
+  Universe documentation.
+
+Dasharo repository contains release tags which are associated with Dasharo
```

Comment:
Why a single bullet point, and then the rest of text in paragraph?

As a user, when seeing a bullet in such question, I would expect a list of bullets, describing differences of Dasharo vs coreboot.

Reply:
I guess we would like to grow that list over time, but for now single bullet doesn't make sense, so I will remove it.

Reply:
Fixed: https://github.com/Dasharo/docs/pull/507/commits/f401d8d1227be7655d44b91669f4dda458cb01d7

Reply:
Thanks. Let's see how much more exceptions we will need.